### ISSUE TITLE
NNFrames support for Tensorflow

### ISSUE THREAD

**Issue by hhbyyh** [intel-analytics/analytics-zoo#1576]
1. check the model training
2. check save/load pipeline
3. provide examples as recommendation for best practice

**Comment by hhbyyh** [comment]
Supporting TensorFlow model with input and prediction nodes are verified. Yet it's not supporting loss function defined with TF.
